tests/DotNetLightning.Infrastructure.Tests/ChannelOperationTests.fs

```diff
@@ -312,8 +337,164 @@ let tests =
+            let! r = bobAcceptedAddHTLCTask
+            Expect.isSome r "timeout waiting for bob to accept htlc"
+
+
+            do! Async.Sleep 1000
+            Console.WriteLine("alice signs her commit for the htlc")
+
+            let aliceAcceptedSign =
+                alice.EventAggregator.AwaitChannelEvent(function WeAcceptedCMDSign (_, commitments) -> Some commitments | _ -> None)
+            let aliceAcceptedRevokeAndAck =
+                alice.EventAggregator.AwaitChannelEvent(function WeAcceptedRevokeAndACK commitments -> Some commitments | _ -> None)
+            let bobAcceptedSign =
+                bob.EventAggregator.AwaitChannelEvent(function WeAcceptedCommitmentSigned (revokeAndAck, commitments) -> Some (revokeAndAck, commitments) | _ -> None)
+            do! alice.CM.AcceptCommandAsync({ NodeId = bobNodeId; ChannelCommand = SignCommitment }).AsTask() |> Async.AwaitTask
+            let! r = aliceAcceptedSign
+            Expect.isSome r "timeout waiting for alice to accept sign command"
+            let! r = bobAcceptedSign
+            Expect.isSome r "timeout waiting for bob to accept commitment signature"
+            let revokeAndAck, bobCommitments = r.Value
+            let! r = aliceAcceptedRevokeAndAck
+            Expect.isSome r "timeout waiting for alice to accept revoke_and_ack"
+            let aliceCommitments = r.Value
+
+
+            do! Async.Sleep 1000
+            Console.WriteLine(sprintf "alice now has: %s sat" (aliceCommitments.LocalCommit.Spec.ToLocal.ToString()))
+
+            let aliceExpectedAmount =
+                LNMoney.Satoshis(fundingSatoshis.Satoshi) - pushMsat - aliceToBobAmount// + bobToAliceAmount
+            let bobExpectedAmount =
+                pushMsat + aliceToBobAmount
+
+            Expect.equal
+                aliceExpectedAmount
+                aliceCommitments.LocalCommit.Spec.ToLocal
+                ""
+            Expect.equal
+                bobExpectedAmount
+                aliceCommitments.LocalCommit.Spec.ToRemote
+                ""
+            Expect.equal
+                bobExpectedAmount
+                aliceCommitments.RemoteCommit.Spec.ToLocal
+                ""
+            Expect.equal
+                aliceExpectedAmount
+                aliceCommitments.RemoteCommit.Spec.ToRemote
+                ""
+
+            Expect.equal
+                bobExpectedAmount
+                bobCommitments.LocalCommit.Spec.ToLocal
+                ""
+            Expect.equal
+                aliceExpectedAmount
+                bobCommitments.LocalCommit.Spec.ToRemote
+                ""
+            Expect.equal
+                aliceExpectedAmount
+                bobCommitments.RemoteCommit.Spec.ToLocal
+                ""
+            Expect.equal
+                bobExpectedAmount
+                bobCommitments.RemoteCommit.Spec.ToRemote
+                ""
+            *)
```

Comment:
can you please create a "revokation WIP PR" where you can add this piece of code above^ (and the `Send mono-hop unidirectional payment with revoke_and_ack` test), to remove it from here

Reply:
This wasn't actually necessary since the test now checks that both peers send the `revoke_and_ack`, so the highly-commented test case was redundant and has been removed.

Checking that revocation secrets are handled correctly would require an entirely different test.